package.json

```diff
@@ -24,7 +27,9 @@
     "prettier:js": "prettier --write './**/*.js'",
     "pretty-check": "prettier --check 'contracts/**/*.sol' && prettier --check './**/*.js'",
     "spread-orders": "npx truffle exec scripts/stablex/place_spread_orders.js",
-    "ts": "yarn run ts-contracts && yarn run ts-check && tsc",
+    "ts": "yarn run ts-contracts && yarn run ts-check && yarn run tsc:commonjs && yarn run tsc:esm",
+    "tsc:commonjs": "rimraf build && tsc",
+    "tsc:esm": "rimraf typescript-esm && tsc -m es6 -t esnext --outDir typescript-esm",
```

Comment:
ubernit: is it possible to have the output be in `typescript/common` and `typescript/esm` so that there is only one directly in the `.gitignore`. IMO its a little cleaner that way.